### PR TITLE
Doc: update getting started guide to support V0.3 release

### PR DESCRIPTION
**Pull request by ailun258**
updates:  
1, update Clear Linux version to 26120 support v0.3 release
2,  update product kernel to kernel-iot-lts2018 since there is new named bundle 

Signed-off-by: Ailun258 <ailin.yang@intel.com>

### CODE REVIEW

**Comment by ailun258** [comment]
@dbkinder @NanlinXie  please help review and merge

**Comment by HuiHuangShi** [comment]
1786 <br>The PR does not touch any hypervisor source file. Skip coding guideline checks.<br>

**Comment by ailun258** [comment]
coding guideline checks is default action, please Nanlin to help fix and merge this. 
@NanlinXie

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -37,13 +37,13 @@ complete this setup.
...
-   ACRN v0.2 (and the current master branch) requires Clear Linux
-   version 25130 or newer.  If you use a newer version of Clear Linux,
+   ACRN v0.3 (and the current master branch) requires Clear Linux
+   version 26120 or newer.  If you use a newer version of Clear Linux,
    you'll need to adjust the instructions below to reference the version
    number of Clear Linux you are using.
 
 #. Download the compressed Clear installer image from
-   https://download.clearlinux.org/releases/25130/clear/clear-25130-installer.img.xz
+   https://download.clearlinux.org/releases/261200/clear/clear-26120-installer.img.xz
```
Comment (dbkinder) [inline doc/getting-started/apl-nuc.rst]: 261200 has an extra 0

Comment (dbkinder) [reply doc/getting-started/apl-nuc.rst]: I'll merge and fix this in another PR

**Pull request merged**